e2e_tests/integration/plan.spec.js

```diff
@@ -37,6 +37,26 @@ describe('Plan output', () => {
     const password = Cypress.config('password')
     cy.connect('neo4j', password)
   })
+  it('displays the expanded details by default and displays/hides details when clicking the plan expand/collapse buttons respectively', () => {
+    cy.executeCommand(':clear')
+    cy.executeCommand(
+      'EXPLAIN MATCH (n:Person) WHERE n.age > 18 RETURN n.name ORDER BY n.age'
+    )
+
+    // initially should be expanded
+    const el1 = cy.get('[data-testid="planSvg"]', { timeout: 10000 })
+    el1.find('.detail').should('exist')
```

Comment:
Im having to do a `.find('.detail')` as the `.detail` classes (e.g. `.detail identifiers` etc) were the only things I could identify in the dom as determining whether expanded or not. I don't really like this and see we don't really use class names in this manner elsewhere but I couldn't think of another way.